microsoft/testsuites/dpdk/dpdkutil.py

```diff
@@ -209,7 +210,13 @@ def initialize_node_resources(
     testpmd.set_dpdk_source(dpdk_source)
     testpmd.set_dpdk_branch(dpdk_branch)
     testpmd.add_sample_apps_to_build_list(sample_apps)
-    testpmd.install()
+    try:
+        testpmd.install()
+    except UnsupportedDistroException as err:
+        # forward message from distro exception, add OS Version info that is removed.
+        raise SkippedException(
```

Comment:
It can totally use the exception message from `UnsupportedDistroException` like `raise SkippedException(err)`.

Reply:
Oh cool! Alright will switch